locale/en_US/submission.xml

```diff
@@ -44,6 +44,7 @@
 	<message key="submission.confirmSubmit">Are you sure you wish to submit this manuscript to the press?</message>
 	<message key="submission.metadata">Metadata</message>
 	<message key="submission.supportingAgencies">Supporting Agencies</message>
+	<message key="submission.grid.titleColumn">Author; Book Title</message.
```

Comment:
I agree but I couldn't find a good place to put the locale key in `pkp-lib`. Do you have any ideas? I struggle to really grasp the scope of the various files and I'm often not sure of the right place to put a locale key... I searched for grid-related strings but didn't find anything that looked appropriate.